src/components/dataset-details-page/index.tsx

```diff
@@ -97,10 +99,18 @@ const DatasetDetailsPage: FC<Props> = ({
   }, [conceptIdentifiers?.join()]);
 
   useEffect(() => {
-    if (datasetUris && datasetUris.length > 0 && datasets.length === 0) {
-      getDatasets({
-        uris: datasetUris.join()
-      });
+    if (
+      datasetId &&
+      datasetUris &&
+      datasetUris.length > 0 &&
+      datasets.length === 0
+    ) {
+      getDatasets(
+        {
+          uris: datasetUris.join()
+        },
+        datasetId
+      );
```

Comment:
Jeg synes at det blir uansett best om vi fyrer av en request hver eneste gang komponenten mountes.

Reply:
Hmm, var det ikke akkurat det vi snakket om at vi ikke må gjøre når vi har redux, da er hele poenget med redux borte og vi trenger ikke innføre redux in the first place. Det er jo ikke poeng å ha redux som sletter hver eneste gang.

Reply:
Vi bruker redux for å ha bedre sporbarhet og kontroll, og ikke nødvendigvis caching. Caching kan være nyttig hvis det er snakk om mega store mengder data, men ikke i vårt tilfellet. Redux er der for å skille data fra GUI.

Reply:
Men vi mister jo mye av sporbarheten hvis man alltid sletter hele store. Ellers kan vi jo skille data og GUI på andre måter, helt uten redux hvis det uansett ikke er poeng å ha. Vi har gjort det på en del andre komponenter, blir mye mindre kompleksitet ut av det hvis man uansett ikke benytter data på tvers.

Reply:
Den alternative måten som du foreslår er at det fyres request selv om det ikke finnes uris, for å slette store. Siden datasetUris er avhengig av annen async request må vi da ha deps [(datasetUris ?? []).join()] og vi ender opp med å sende to requests, en tom uris (før dataset er resolved), og etterpå en med uris når datasett er resolved.